docs/src/pages/customization/theming/CustomStyles.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '@material-ui/core/styles';
 import { orange } from '@material-ui/core/colors';
 
-declare module '@material-ui/core/styles/createTheme' {
+declare module '@material-ui/core/styles/createMuiTheme' {
```

Comment:
I think this can be 
```suggestion
declare module '@material-ui/core/styles' {
```
now @mnajdova?

Reply:
Yes, this should work.

Reply:
Yes, I agree.